portal/models.py

```diff
@@ -14,13 +13,48 @@
     PageChooserPanel,
 )
 from wagtail.fields import RichTextField, StreamField
-from wagtail.models import Orderable, Page
 from wagtail.images.models import Image
+from wagtail.models import Orderable, Page
 from wagtail.search import index
 
+from streams import blocks
+
 from . import get_collections
 
 
+class PortalBasePage(Page):
+    """Base page class for BMRC portal pages"""
+    """
+    This makes it easier to pass on these dictionaries
+    for multiple portal pages.
+    These are useful for navigation and facets. """
```

Comment:
This should normally be a comment not a docstring (or just move it inside the docstring).